templates/deployments/appname/app.jsonnet.tpl

```diff
@@ -62,7 +62,11 @@ local all = {
 	namespace: ok.Namespace(app.namespace) {
 		metadata+: {
 			annotations+: {
-				'iam.amazonaws.com/permitted': '%s_service_role' % app.name,
+        {{- if not (stencil.Arg "aws.useKIAM") }}
+				'eks.amazonaws.com/role-arn': 'arn:aws:iam::'+ {{ stencil.Arg "aws.account" }} + ':role/' + app.bento + '-' + app.name
```

Comment:
thought: it might be easier to read as an interpolated string rather than a concatenated one.

Reply:
Agree here, meant to change it to interpolated. Will do that!